env.example

```diff
@@ -68,7 +68,7 @@ MOCK_ELASTIC=
 ELASTIC_URL=http://127.0.0.1
 ELASTIC_PORT=9200
 # Max number of results per query
-ELASTIC_MAX_RESULTS=50
+ELASTIC_MAX_RESULTS_PER_PAGE=5
```

Comment:
Let's make this same change to https://github.com/Seneca-CDOT/telescope/blob/master/env.production and https://github.com/Seneca-CDOT/telescope/blob/master/env.staging.

Reply:
done